### PR TITLE
moving more tags

### PR DESCRIPTION
**Pull request by arwilczek90**
made some minor code changes which added more tags to be removed so I refactored that process and added the worker_guid as a top level field

### CODE REVIEW

File: logstash_formatter/__init__.py [hunk 1]
```diff
@@ -84,32 +84,28 @@ def format(self, record):
...
 
-        if 'msg' in fields:
-            fields.pop('msg')
-
-        if 'exc_info' in fields:
-            if fields['exc_info']:
-                formatted = tb.format_exception(*fields['exc_info'])
-                fields['exception'] = formatted
-            fields.pop('exc_info')
+        if fields['exc_info']:
```
Comment (dennismonsewicz) [inline logstash_formatter/__init__.py]: do you still want to check for `exc_info` in the fields dictionary? If this doesn't exist, it'll throw a key error

File: logstash_formatter/__init__.py [hunk 2]
```diff
@@ -84,32 +84,28 @@ def format(self, record):
...
 
-        if 'msg' in fields:
-            fields.pop('msg')
-
-        if 'exc_info' in fields:
-            if fields['exc_info']:
-                formatted = tb.format_exception(*fields['exc_info'])
-                fields['exception'] = formatted
-            fields.pop('exc_info')
+        if fields.get('exc_info', None):
```
Comment (dennismonsewicz) [inline logstash_formatter/__init__.py]: You don't have to set `None` as a default here. `if fields.get('exc_info')` should return None if it's not there

``` python
In [1]: fields = {}

In [2]: if fields.get('exc_info'):
   ...:     print 'yolo'
   ...:

In [3]: fields = {'exc_info': 'yolo'}

In [4]: if fields.get('exc_info'):
    print 'yolo'
   ...:
yolo

```

Comment (arwilczek90) [inline logstash_formatter/__init__.py]: thats right I was thinking of dict.pop where if you don't supply the default it throws an exception.

**Comment by dennismonsewicz** [comment]
👍 great work bud

**Pull request merged**